docs/docusaurus/docs/deployment_patterns/how_to_use_gx_with_aws/how_to_use_gx_with_aws_using_athena.md

```diff
@@ -295,7 +295,7 @@ This guide will demonstrate each of the steps necessary to go from installing a
 
 <ConfigureYourDatasource />
 
-Datasources can be configured in many customized ways.  For additional information on how to configure a SQL datasource such as the one used to connect to your Athena data, please see our guide on [how to configure a SQL Datasource](../../../docs/guides/connecting_to_your_data/datasource_configuration/how_to_configure_a_sql_datasource.md)
+Datasources can be configured in many customized ways.  For additional information on how to configure a SQL datasource such as the one used to connect to your Athena data, please see our guide on [how to configure a SQL Datasource](/docs/0.15.50/guides/connecting_to_your_data/datasource_configuration/how_to_configure_a_sql_datasource)
```

Comment:
we have a new doc for this, why are you linking to v15?

Reply:
My understanding is that we have not yet updated the "deployment pattern" guides to be fluent datasource compatible.  So, I linked to the block-config version of the docs so that we aren't having people cross between fluent and block config datasources again when they look for their reference materials.

Reply:
updated